src/libs/actions/PersonalDetails.js

```diff
@@ -433,4 +472,5 @@ export {
     updateProfile,
     updateDisplayName,
     clearAvatarErrors,
+    buildNewPersonalDetails,
```

Comment:
I am not sure if we should call this `buildNewPersonalDetails`. This could be `buildOptimisticPersonalDetails` ?